models/membership_test.go

```diff
@@ -58,6 +59,62 @@ var _ = Describe("Membership Model", func() {
 				Expect(dbMembership.ClanID).To(Equal(membership.ClanID))
 			})
 
+			It("Should get existing Membership", func() {
+				_, _, _, _, memberships, err := fixtures.GetClanWithMemberships(testDb, 0, 0, 0, 1, "", "")
```

Comment:
(future suggestion) we should name each var passed to this fixture, to make clear what we are creating here.

Reply:
Yeah, this function is horrible